src/bertron_schema/schema/bertron_schema.yaml

```diff
@@ -1,96 +1,288 @@
----
 id: https://w3id.org/ber-data/bertron-schema
 name: bertron-schema
-title: bertron-schema
-description: |-
-  LinkML schema for ber data intergration work
+title: BERtron schema
+description: Schema for BERtron common data model.
+version: 0.0.1
 license: BSD-3
 see_also:
   - https://ber-data.github.io/bertron-schema
-
 prefixes:
-  bertron_schema: https://w3id.org/ber-data/bertron-schema/
+  bertron: https://w3id.org/ber-data/bertron-schema/
   linkml: https://w3id.org/linkml/
-  biolink: https://w3id.org/biolink/
   schema: http://schema.org/
-  PATO: http://purl.obolibrary.org/obo/PATO_
-  example: https://example.org/
-default_prefix: bertron_schema
+default_prefix: bertron
 default_range: string
-
+default_curi_maps:
+  - semweb_context
 imports:
-  - linkml:types
+  - bertron_types
 
 classes:
 
-  NamedThing:
-    description: >-
-      A generic grouping for any identifiable entity
-    slots:
-      - id
-      - name
-      - description
+  Entity:
     class_uri: schema:Thing
+    description: |
+      An object retrieved by BERtron from a BER data API.
+    attributes:
+      ber_data_source:
+        description: The BER member from whence the entity originated.
+        range: BERSourceType
+        required: true
+      # TODO!
+      credit_metadata:
+        description: Citation metadata for the entity.
+        todos:
+        - add in KBase credit metadata schema
+      coordinates:
+        description: The geographic coordinates associated with an entity. For entities with a bounding box, the centroid is used as the geographic reference.
+        range: Coordinates
+        required: true
+      data_type:
+        description: What kind of data this entity is -- e.g. sequence data; a soil core; a well; etc.
+        range: DataTypeTagType
+        multivalued: true
+        required: true
+      description:
+        description: Textual description of the entity.
+        required: false
+        slot_uri: schema:description
+        examples:
+        - value: "River water sample taken by AquaTROLL 9000."
+        - value: "Genome sequence of P. aeruginosa strain IDDQD"
+      id:
+        aliases:
+        - BER data source internal identifier
+        - CURIE
+        description: The unique ID used for the entity within the BER resource. It may not necessarily be resolvable outside the resource.
+        comments:
+        - If the data source does not use CURIEs, we cannot guarantee that IDs will be unique between all the BER sources.
+        # identifier: true
+        # required: true
```

Comment:
Ideally `identifier` and `required` should both be true here -- this field should provide a unique ID amongst results returned by the BERtron search.